client/src/AppContainer.tsx

```diff
@@ -1,15 +1,14 @@
 import React, { useState, useEffect } from 'react';
-import Keycloak from 'keycloak-js';
-import { KeycloakProvider } from '@react-keycloak/web';
 import { ApolloOfflineClient } from 'offix-client';
 import { ApolloOfflineProvider } from 'react-offix-hooks';
 import { ApolloProvider } from '@apollo/react-hooks';
 import { AppContext } from './AppContext';
-import { keycloakConfig, clientConfig } from './config';
-import { keycloakEnabled, onKeycloakTokens } from './helpers/keycloakHelpers';
+import { clientConfig } from './config';
 import { Loading } from './components/Loading';
 import { IContainerProps } from './declarations';
+import { getKeycloakInstance } from './auth/keycloakAuth';
 
+let keycloak: any;
 const client = new ApolloOfflineClient(clientConfig);
```

Comment:
In this context naming this graphqlClient or apolloClient will be much better. There is keycloak and other things happening here